plugins/mcreator-core/jsontriggers/player_breed_animals_advanced.json

```diff
@@ -0,0 +1,25 @@
+{
+  "args0": [
+    {
+      "type": "field_data_list_selector",
+      "name": "parent",
+      "datalist": "entity"
+    },
+    {
+      "type": "field_data_list_selector",

```

Comment:
I would also personally reemove this block and simply use the first block to create this behaviour. For example, as this would be a list, if there's only one entry, the first block' s behaviour is used. If there's 2 or more entities (so the "any of" behaviour), you take the first entity for the parent parameter and the other ones go inside the partner parameter.

Another option could also be to change this block, so both parameter have an independant "Any of"/repeating params field. This would also avoid some cases users don't want to get, so the second option is much more preferable.

This trigger would be much more useful if added only after player and entity conditions where added. This would also avoid work and converters.